src/components/LocationPage/LocationPageHeader.tsx

```diff
@@ -198,13 +167,14 @@ const LocationPageHeader = (props: {
                 <Copy>{levelInfo.detail(locationName)}</Copy>
               </SectionColumn>
             </SectionHalf>
+            {/* TODO: Re-add this if we have new updates in the future.
```

Comment:
I left this commented-out in case it's handy again in the future but I'm also happy to just rip it out and we can dig it out of git if/when we need it.

Reply:
hmm my vote is for delete and we can dig it out if/when needed